kensu/google/cloud/bigquery/job/offline_parser.py

```diff
@@ -52,16 +93,24 @@ def get_referenced_tables_metadata(
             metadata["tables"].append(table_md)
         return metadata,  table_id_to_bqtable, table_infos
 
+
+    @staticmethod
+    def table_ref_to_kensu(
+            client: bq.Client,
+            table_id: Union[Table, TableReference, str],
+    ):
+        table = client.get_table(table_id)
+        ds, sc = BqKensuHelpers.table_to_kensu(table)  # FIXME?
```

Comment:
Fix what?

Reply:
very good question. no clue :D I think I'll just need to delete this comment...

P.S. this is old, originally from here: https://github.com/kensuio-oss/kensu-py/pull/83/files#diff-8a93f78e33c7985d19028f1e87110efb62b9bce1074e09b1d66e12fbc9ad3b0cL62-L64